dipy/tracking/tests/test_streamline.py

```diff
@@ -516,6 +517,63 @@ def test_unlist_relist_streamlines():
         assert_array_equal(streamlines[i], streamlines2[i])
 
 
+def test_deform_streamlines():
+    # streamlines needs to be a list of streamlines
+    A = np.array([[1, 2, 3], [1, 2, 3.]])
+    streamlines = [A for i in range(10)]
+
+    # Create Random deformation field
+    deformation_field = np.empty((8, 8, 8, 3))
```

Comment:
You can generate a random 4D numpy arrays like this `deformation_field = np.random.randn(8, 8, 8, 3)`.

Reply:
sounds good, changed.